ompi/mpi/man/man3/MPI_Win_shared_query.3in

```diff
@@ -82,6 +82,10 @@ all processes in the group attached to the window specified \fIsize\fP
 = 0, then the call returns \fIsize\fP = 0 and a \fIbaseptr\fP as if
 \fBMPI_Alloc_mem\fP was called with \fIsize\fP = 0.
 
+.SH C NOTES
+.ft R
+While \fIbaseptr\fP is a \fIvoid *\fP type, this is to allow easy use of any pointer object for this parameter. This argument is really a \fIvoid **\fP type.
```

Comment:
```suggestion
The parameter \fIbaseptr\fP is of type \fIvoid *\fP  in C to allow passing of any pointer object for this parameter. This argument should be a pointer to a pointer of arbitrary type (e.g., \fIvoid **\fP).
```

Reply:
I don't think mentioning C under `C notes` is necessary? Right?